res/database/Default/user/scr0llbaer.xml

```diff
@@ -700,8 +700,6 @@ Format: Filmowane wykłady. Na luzie i do leżenia.</pl>
 		<scripttemplate product="4" licence_type="1" guid="494235d3-660d-4214-a687-4a5d0054943b" creator="9551" comment="Perhaps some reciprocal effect with the corresponding news genre can be implemented? TODO: I am not happy with the titles and descriptions of these.">
 			<title>
 				<de>${tower} ${latebreaking}: ${title}</de>
```

Comment:
what do you mean?

Reply:
ah, `<all>`, yes thanks, I fixed those in other places.

Reply:
Yes, I meant "all". In the editor validation preferences, you can enable "localization duplicates". This will mark places where all language entries are identical which allows you to check whether this is intended or could be replaced by "all" or a language independent variant.